docs/_articles/en/v3.x/faq.md

```diff
@@ -21,7 +21,7 @@ A: No! `Salesforce Code Analyzer` can be used on any codebase.
 ## Questions about language support
 
 #### Q: What languages does `Salesforce Code Analyzer` support?
-A: By default, `Salesforce Code Analyzer` supports code written in Apex, VisualForce, Java, JavaScript, XML, and TypeScript.
+A: By default, `Salesforce Code Analyzer` supports code written in Apex, VisualForce, Java, JavaScript, XML, and TypeScript. You can specifically invoke `scanner:run` command with `--engine eslint-lwc` to get support for LWC.
```

Comment:
Unrelated to this change. I'm addressing the issue in W-11294404 since it seemed too trivial to handle separately.

Reply:
Could you make this its own PR? I know it's a small change, but it's better for our bookkeeping if work records have their own PRs.

Reply:
I included this here to squeeze in the change given the short time we have. If we need the bookkeeping, I'd rather include this with Teresa's other changes and not have it here.

Reply:
Fair enough.